### ISSUE TITLE
Faders are resisting

### ISSUE THREAD

**Issue by RayesMusic** [bramdebouvere/fltouch#20]
**Hello!**
I've been using your script for my X-touch for a few months now. And it always worked. But, recently, I just bought a new MIDI controller... Since I started using it, the xtouch never worked normally again, even after resetting everything and removing the other device. I never changed any settings.

**The problem is:**
-The faders are resisting to my movement in every situation, even in the "free mode" impossible to use it then.

**What I tried:**
-Updating the xtouch firmware
-Trying in others DAWs -> _works perfectly_
-Trying the official Mackie control script from Image-Line -> _Still the same problem_
-Trying to delete the USB bus port in Windows settings and reinstall those
-Trying to change the UBS cable, and plugging it in different USB ports
-Fully re-install FL STUDIO with stock settings

I was thinking that maybe disabling the motors will change something... but I don't know how

I have no idea what is going on, or if it's your script or not. If you have any suggestion, or ideas, it would save my life!

**Regards,
RAYES**

**Comment by bramdebouvere** [comment]
Hey there,

Since the official Mackie control script seems to have the same issue, I doubt that the script will be to blame.

Perhaps the issue lies in one of the following?
- Incorrect port number on in or output? (might have changed when you added the new midi controller)
- Software from your new MIDI controller interfering with the MIDI info?

I'd suggest checking these out. Please let me know if you're able to fix the issue?

I cannot tell for certain what's wrong. Perhaps it's also worth it to check if you get any errors in the _View -> Script Output_ window?

**Comment by RayesMusic** [comment]
Hey, 
thank you for your answer! 

I have the MIDI assignments in FL STUDIO which are set as in the instructions.
As for the external midi controller software, it's plug and play and I uninstalled the software associated with it.

it's an incomprehensible problem that I've been struggling to solve for more than two weeks.

I also checked in FL STUDIO if the script output was correct an it is.
I think i will try the setup on another computer to see if it solves the problem.

I'll keep you informed if i find a solution,

RAYES
